pkg/apis/core/validation/shoot.go

```diff
@@ -169,6 +171,10 @@ func ValidateShootTemplateUpdate(newShootTemplate, oldShootTemplate *core.ShootT
 
 	allErrs = append(allErrs, ValidateShootSpecUpdate(&newShootTemplate.Spec, &oldShootTemplate.Spec, newShootTemplate.ObjectMeta, fldPath.Child("spec"))...)
 
+	if oldShootTemplate.Spec.Networking.Nodes != nil {
+		allErrs = append(allErrs, apivalidation.ValidateImmutableField(newShootTemplate.Spec.Networking.Nodes, oldShootTemplate.Spec.Networking.Nodes, fldPath.Child("spec", "networking", "nodes"))...)
+	}
```

Comment:
Why don't we use the feature gate also here? 😕

Reply:
Is this question answered by the comments below?
Or do you see any risk in not using the feature gate here?